Source/Core/Core/IOS/USB/Bluetooth/LibUSBBluetoothAdapter.cpp

```diff
@@ -77,6 +61,21 @@ bool LibUSBBluetoothAdapter::IsConfiguredBluetoothDevice(u16 vid, u16 pid)
   return configured_vid == vid && configured_pid == pid;
 }
 
+bool LibUSBBluetoothAdapter::IsBluetoothDevice(const libusb_device_descriptor& descriptor)
```

Comment:
Nitpick: If this function definition was moved up to like line 45 the diff would be less noisy.